src/test/java/LoginTest.java

```diff
@@ -0,0 +1,24 @@
+import pages.HomePage;
+import pages.LoginPage;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class LoginTest extends BaseTest{
+
+    @Test
+    public void loginWithCorrectCredentials(){
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
+
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(2));
```

Comment:
Delete `WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(2));`

Variable 'wait' is never used